tests/cypress/types/tasks.d.ts

```diff
@@ -0,0 +1,17 @@
+/// <reference types="cypress" />
+
+import type {
+  CreateVirtualTestnetOptions,
+  CreateVirtualTestnetResponse,
+  DeleteVirtualTestnetsOptions,
+} from 'support/helpers/tenderly'
+
+/** Doesn't work properly, because arg is still inferred as `any`. Not sure if this is solvable, leaving it here for now */
+declare global {
+  namespace Cypress {
+    interface Chainable {
+      task(event: 'createVirtualTestnet', arg: CreateVirtualTestnetOptions): Chainable<CreateVirtualTestnetResponse>
```

Comment:
Does this really need be a task that runs in nodejs? I think it can run in the browser just fine, it's just fetch requests